app/assets/stylesheets/thyme.scss

```diff
@@ -341,3 +418,234 @@ figure {
  {
    background-color: black;
  }
+
+
+
+/* annotation modal */
+#annotation-modal {
+  .modal-content {
+    width: 100%;
+  }
+  #annotation-modal-content {
+    width: 100%;
+  }
+  #preview-toggle {
+    input:checked + .slider {
+      background-color: #0D6EFD;
+    }
+    .switch {
+      vertical-align: middle;
+    }
+  }
+  #annotation-modal-preview {
+    height: 543px;
```

Comment:
Don't hardcode variables like that. Instead use relative positioning and specify how this element should behave with respect to other elements, e.g. "It should have the same height as this other element" (use case for `height: 100%`). Right now, for long notes, this is the reason why the preview is cut off here:

<img width="646" alt="image" src="https://github.com/MaMpf-HD/mampf/assets/37160523/15b56dd9-cfd5-4b70-9c4a-e0af137b3714">